demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -243,6 +250,15 @@ def dump(self, path: Path, marketplace: MarketplaceVersions):
                 shutil.copy(self.path / "Author_image.png", path / "Author_image.png")
             except FileNotFoundError:
                 logger.info(f'No such file {self.path / "Author_image.png"}')
+            if self.object_id == "Base":
```

Comment:
please move to sep method handle_base_pack_documentation_files() and use constants